arbiter/drivers/onedrive.cpp

```diff
@@ -211,7 +146,15 @@ std::vector<std::string> OneDrive::processList(std::string path, bool recursive)
 
         //check for link to next set
         if (obj.contains("@odata.nextLink"))
+        {
             pageUrl = obj.at("@odata.nextLink");
+            const http::Query parsedQueries(http::getQueries(pageUrl));
+
+            for (auto& it: parsedQueries)
```

Comment:
Do we want to keep any query parameters from the previous iteration or should this be a complete replacement?  If we want to replace then simply `queries = http::getQueries(pageUrl)` should suffice.  Otherwise maybe add a comment stating why we're aggregating new queries each time.